spec/acceptance/build_stubbed_spec.rb

```diff
@@ -145,8 +145,9 @@
     ENV['TZ'] = original_timezone
   end
 
-  it "adds created_at to objects who don't have the method" do
-    expect(build_stubbed(:thing_without_timestamp)).to respond_to(:created_at)
+  it "doesn't add created_at to objects who don't have the method" do
```

Comment:
Is looks like this behavior was intentional. @joshuaclayton do you remember the context for adding `created_at` to an object that wouldn't normally respond to that? This was added in 4bccbb90d1f4bfe8e55bb48f4e93a760f77d1b8f

Reply:
It's the result of https://github.com/thoughtbot/factory_bot/pull/390, where there's further discussion. Looking at this now, I agree that defining these methods automatically is a bit odd, as it's basing that assumption on FB being used only for ORM-backed classes.

The ability to override `created_at`/`updated_at`, however, is something I think we'd want to retain in cases where the class does inherit from `AR::B`/`ApplicationRecord`/et al, so this should be good to go. I still can't find a reference where we determine a need to set it on objects that don't define `created_at` though.